config/secrets.yml

```diff
@@ -115,12 +115,13 @@ default: &default
   custom_conference_types:
     - key: parallel # used to find the I18n key and route path
       url: /conferences/parallel
```

Comment:
```suggestion
```
This probably shouldn't be needed. We should use a different approach instead of using this url directly (than can collide with conferences with the same slug).
We should refactor to use a query string instead using the hashtag key:
`/conferences?type=activity`